src/components/index.js

```diff
@@ -0,0 +1,3 @@
+import Footer from "./Footer";
+
+export default Footer;
```

Comment:
import Footer from "./footer" this is because footer is not a component here, but a path. Take note